src/main/webapp/mentor_form.html

```diff
@@ -11,14 +11,15 @@
   <h1 class="display-4 mt-5">Become a Mentor for Your Project!</h1>
   <h4 class="emphasis mb-5">Enter the URL of the GitHub repository for your project to become a mentor.</h4>
   <hr/>
-  <form>
+  <form action="javascript:submitForm()">
```

Comment:
nit: In the spirit of [separation of concerns from the style guide](https://google.github.io/styleguide/htmlcssguide.html#Separation_of_Concerns), you could add this via a [submit event listener instead](https://developer.mozilla.org/en-US/docs/Web/API/HTMLFormElement/submit_event): 
```javascript
form.addEventListener('submit', submitForm());
```